Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook

```diff
@@ -6,69 +6,6 @@
       "content": {
         "version": "KqlParameterItem/1.0",
         "parameters": [
-          {
-            "id": "c1351831-278e-4a4d-a8e4-b9f45361abc2",
-            "version": "KqlParameterItem/1.0",
-            "name": "timeRange",

```

Comment:
you might want time range on every tab, especially with logs.  some customers have huge amounts of data, and a hardcoded "1 day" or whatever might fail for them?

Reply:
There was a specific request for me to move the time range selector out because the data being shown on the other 2 tabs is either already filtering down to the smallest set of data or a specific case that requires a fixed time range